components/published-toggle/index.jsx

```diff
@@ -19,12 +19,8 @@ const PublishedToggle = ({
 
   const handleChange = useCallback(async event => {
     const isVisible = event.target.checked
-    try {
-      await onVisibilityChanged(isVisible)
-      toggleChecked(isVisible)
-    } catch (e) {
-      // something happened, don't update state
-    }
+    await onVisibilityChanged(isVisible)
```

Comment:
What do you get back after actual changing `disable` property?

I imagine that:
1. you press a button
2. request is sent
3. store gets new entity in the response
4. store updates the entity
5. MobX triggers observers to update
6. React gets updated

Reply:
That's what I wanted to do at first but it's not that easy to do it.